secure_message/application.py

```diff
@@ -71,11 +72,15 @@ def create_app(config=None):
     api.add_resource(MessageSendV2, '/v2/messages')
     api.add_resource(MessageCounterV2, '/v2/messages/count')
 
+    app.token_expires_at = maya.now()
```

Comment:
This isn't needed as you're doing it in the cache_client_token function.

Reply:
I wanted to have the attribute set on the app at startup